src/sentry/api/endpoints/project_group_index.py

```diff
@@ -604,13 +658,21 @@ def put(self, request, project):
                         if not is_bulk:
                             activity.send_notification()
 
-                issue_resolved_in_release.send_robust(
-                    group=group,
-                    project=project,
-                    user=acting_user,
-                    resolution_type=res_type_str,
-                    sender=self.__class__,
-                )
+                if release:
+                    issue_resolved_in_release.send_robust(
```

Comment:
this was incorrectly firing all of the time for any resolution

Reply:
this was intentional, we wanted to capture manual resolutions too (this broke amplitude stats and feature adoption table so I noticed). can you elaborate what was incorrect?

Reply:
we should probably call it something else then (vs `in_release`) as it wasn't clear of that intention